sessions/15/pavel_shchegolevatykh/default_dict.py

```diff
@@ -0,0 +1,31 @@
+class MyDefaultDict(dict):
+
+    def __init__(self, *args, **kwargs):
+        if 'default' in kwargs:
+            self.default = kwargs['default']
```

Comment:
you can simplify it all by just:
kawrgs.pop('default', None)

Reply:
Very good to know. Thanks! Fixed.